docs/overlay.md

```diff
@@ -1,6 +1,146 @@
-Overlay Driver
-==============
+# Multi-Host Overlay Driver Quick Start
 
-## Configuration
+This example is to provision two Docker Hosts with the **experimental** Libnetwork default overlay network driver.
+
+### Pre-Requisites
+
+- Kernel >= 3.1.6
+- Install Docker Experimental
+- Experimental Docker client
+
+### Install Docker Experimental
+
+Follow Docker experimental installation instructions at: [https://github.com/docker/docker/tree/master/experimental](https://github.com/docker/docker/tree/master/experimental)
+
+To ensure you are running the experimental Docker branch, check the version and look for the experimental tag:
+
+```
+$ docker -v
+Docker version 1.8.0-dev, build f39b9a0, experimental
+```
+
+### Install and Bootstrap K/V Store
+
+
+Multi-host networking uses a pluggable Key-Value store backend to distribute states using `libkv`.
+`libkv` supports multiple pluggable backends such as `consul`, `etcd` & `zookeeper` (more to come).
+
+In this example we will use `consul`
+
+Install:
+
+```
+$ curl -OL https://dl.bintray.com/mitchellh/consul/#0.5.2_linux_amd64.zip
+$ unzip 0.5.2_linux_amd64.zip
+$ mv consul /usr/local/bin/
+```
+
+**host-1** Start Consul as a server in bootstrap mode:
+
+``` 
+$ consul agent -server -bootstrap -data-dir /tmp/consul -bind=<host-1-ip-address>`
+```
+
+**host-2** Start the Consul agent:
+
+``` 
+$ consul agent -data-dir /tmp/consul -bind <host-2-ip-address>
+$ consul join <host-1-ip-address>
+```
+
+
+### Start the Docker Daemon with the Network Driver Daemon Flags
+
+**host-1** Docker daemon:
+
+```
+$ docker -d --kv-store=consul:localhost:8500 --label=com.docker.network.driver.overlay.bind_interface=eth0
+```
+
+**host-2** Docker daemon needs to point to member of the Consul cluster. The following example starts the Docker daemon in the foreground. The global flags can be added to the docker configuration file:
+
+```
+$ docker -d --kv-store=consul:localhost:8500 --label=com.docker.network.driver.overlay.bind_interface=eth0 --label=com.docker.network.driver.overlay.neighbor_ip=<host-1-ip-address>
+```
+
+### QuickStart Containers Attached to a Network
+
+**host-1** Start a container on the network `dev` with a service name of `svc1`
```

Comment:
Would you mind replacing the above text with -

Start a container that publishes a service `svc1` in the network `dev` that is managed by `overlay` driver